signbank/dictionary/tasks.py

```diff
@@ -83,6 +83,11 @@ def retrieve_videos_for_glosses(video_details: List[VideoDetail]):
     s3 = boto3.client("s3")
     s3_storage_used = settings.GLOSS_VIDEO_FILE_STORAGE == "storages.backends.s3boto3.S3Boto3Storage"
 
+    # Set headers for urlretrieve()
+    opener = build_opener()
+    opener.addheaders = settings.NZSL_SHARE_HEADERS
```

Comment:
suggestion: I think we should just have the headers here

```suggestion
    opener.addheaders = [('Accept', '*/*')]
```

as we're not using them anywhere else, and unlikely to need to be changed - even if we were to use them elsewhere, I'd argue we should be doing so via a service that handles the actual URL requests so we'd still only have one place in the code where the headers would be used